.circleci/config.yml

```diff
@@ -69,60 +72,155 @@ commands:
           command: |
             conda create -n hydra python=<< parameters.py_version >> pywin32 -qy
             conda activate hydra
-            pip install nox dataclasses
+            pip install nox@git+git://github.com/theacodes/nox.git@8dcefbf8f59293bc1925dd13376dd6d3aa66e53a
```

Comment:
The cache tag should be a function of the things you are installing.
If we change the nox version here it will not get picked up until the cache expires.

Reply:
That's a good point! I'm not sure how to do this in a more clean way. For now I defined a parameter for cachekey. Everytime we update any dependency, we will need to bump the cachekey.

Reply:
The nox version with --add-timestamp was released, you can install nox normally.